Marlin/src/lcd/e3v2/proui/menus.cpp

```diff
@@ -107,7 +107,7 @@ void onDrawMenuItem(MenuItem* menuitem, int8_t line) {
   if (menuitem->icon) DWINUI::drawIcon(menuitem->icon, ICOX, MBASE(line) - 3);
   if (menuitem->frameid)
     dwinFrameAreaCopy(menuitem->frameid, menuitem->frame.left, menuitem->frame.top, menuitem->frame.right, menuitem->frame.bottom, LBLX, MBASE(line));
-  else if (menuitem->caption)
+  else
```

Comment:
Is it true that `menuitem` always has a `caption`?

Reply:
I think so. almost every instance uses one, this is for using Chinese characters, right? so if it doesnt have a "frameid" then it uses a caption, no?

Reply:
if you really wanted to what if you had it something like
```y
else if (strlen(menuitem->caption))
```
so that if `caption` has any value, it writes a caption. because what happens is for Chinese, `frameid` is true (>0), anytime its not chinese, `frameid` is always 0...false